### DISCUSSION TITLE
SmartAutoClicker-2.1.3

### DISCUSSION THREAD

**Discussion by Nain57** [Nain57/Smart-AutoClicker#282] [Announcements]
# Bug fixes
* The debug overlay/report feature was broken with 2.1.2, it is now fixed.
* Various stabilizations improvements

# Links
[F-Droid](https://f-droid.org/packages/com.buzbuz.smartautoclicker/)
[PlayStore](https://play.google.com/store/apps/details?id=com.buzbuz.smartautoclicker)

<hr /><em>This discussion was created from the release <a href='https://github.com/Nain57/Smart-AutoClicker/releases/tag/2.1.3'>SmartAutoClicker-2.1.3</a>.</em>

**Comment by Someoneyoumightdontknow** [comment]
Hey!!! Thank you for uploading this for the community, i wanted to ask you, how do i use it?
